make/e2e-setup.mk

```diff
@@ -374,10 +374,12 @@ e2e-setup-samplewebhook: load-bin/downloaded/containers/$(CRI_ARCH)/samplewebhoo
 		samplewebhook make/config/samplewebhook/chart >/dev/null
 
 .PHONY: e2e-setup-projectcontour
-e2e-setup-projectcontour: load-$(call image-tar,projectcontour) make/config/projectcontour/contour-gateway.yaml make/config/projectcontour/gateway.yaml bin/scratch/kind-exists bin/tools/kubectl
+e2e-setup-projectcontour: load-$(call image-tar,projectcontour) make/config/projectcontour/contour-gateway.yaml make/config/projectcontour/gateway.yaml bin/scratch/kind-exists bin/tools/kubectl bin/tools/ytt
 	@$(eval SERVICE_IP_PREFIX = $(shell bin/tools/kubectl cluster-info dump | grep -m1 ip-range | cut -d= -f2 | cut -d. -f1,2,3))
-	sed 's|{CLUSTER_IP}|$(SERVICE_IP_PREFIX).14|' make/config/projectcontour/contour-gateway.yaml | bin/tools/kubectl apply -f- >/dev/null
-	bin/tools/kubectl apply -f make/config/projectcontour/gateway.yaml >/dev/null
+	bin/tools/ytt --data-value service_ip_prefix="${SERVICE_IP_PREFIX}" \
```

Comment:
Probably in future all this code could live in separate bash scripts that the make targets could call. That would make it easier to understand the contents of this make file and debug the scripts.

Reply:
Yes, I think aiming to have the make targets simple (and delegate the complexity to bash scripts and preferably using higher level tools) could be one for the technical guidelines (?) doc that you talked about some time ago. I think now would be a good time to make that?